server/.env.example

```diff
@@ -21,3 +21,4 @@ SIGN_IN_PREFILLED=true
 # SUPPORT_DRIVER=front
 # SUPPORT_FRONT_HMAC_KEY=replace_me_with_front_chat_verification_secret
 # SUPPORT_FRONT_CHAT_ID=replace_me_with_front_chat_id
+# SENTRY_URL=https://xxx@xxx.ingest.sentry.io/xxx
```

Comment:
@brendanlaschke Instead of `SENTRY_URL` I think it's better to name it `SENTRY_DSN`, same name as Sentry use is his own library